shared/src/enums/exception-message.enum.ts

```diff
@@ -2,6 +2,7 @@ enum ExceptionMessage {
     INVALID_CREDENTIALS = 'You entered an incorrect e-mail address or password',
     EMAIL_ALREADY_EXISTS = 'User with this email already exists',
     INVALID_TOKEN = 'Invalid token',
+    JWT_MUST_BE_PROVIDED = 'jwt token must be provided',
```

Comment:
no, authorize error

Reply:
Resolved in [befbf22](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/280/commits/befbf22548c59eab40a7bfa1f21aca81523a3939)